### DISCUSSION TITLE
[Bug] second cpu

### DISCUSSION THREAD

**Discussion by lexasub** [RainbowMiner/RainbowMiner#1686] [Q&A] [answered]
rainbowminer didn't see second cpu. how fix it?

**Comment by RainbowMiner** [comment]
Is this a Windows machine? Please create and upload a Debug file.

**Comment by lexasub** [comment]
linux. [debug.zip](https://github.com/RainbowMiner/RainbowMiner/files/6342703/debug.zip)
without gpu logs

**Comment by RainbowMiner** [comment, marked as answer]
Ah ok - since it's a linux machine, things are a bit different. Linux doesn't see multiple CPUs, but instead combines all cores/threads into one "CPU". So, if you happen to have two CPUs with 6 core/12 threads each, Linux will handle these as one CPU with 12 core/24 threads all together. 
RainbowMiner will address the cores/threads correctly, even though it will report "1 CPU found" no matter how many CPUs are installed on the motherboard.

**Comment by lexasub** [comment]
ok, thank you. i close it.
